src/components/SCAStatusSwitch/SCAStatusSwitch.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { FC } from 'react';
+import { Spinner, Switch } from '@patternfly/react-core';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import useUpdateManifestSCAStatus from '../../hooks/useUpdateManifestSCAStatus';
+import './SCAStatusSwitch.scss';
+
+interface SCAStatusSwitchProps {
+  scaStatus: string;
+  uuid: string;
+}
+
+const SCAStatusSwitch: FC<SCAStatusSwitchProps> = ({ scaStatus, uuid }) => {
+  const isChecked = scaStatus.toLowerCase() === 'enabled';
+
+  const { data, mutate, isLoading, isError, isSuccess } = useUpdateManifestSCAStatus();
+
+  const hasError = isError === true || (isSuccess && !data);
```

Comment:
Maybe this could be named "hasProblem" or "failed" or something, just so it's not named exactly like a subset of issues it contains.

Reply:
ok updated that variable name.